### DISCUSSION TITLE
subdomain for custom domain

### DISCUSSION THREAD

**Discussion by alita-moore** [simple-login/app#745] [Feature Requests]
I would like to be able to create different accounts for a given website, and I would like those to route differently. For example, if I want to have a personal and a work account for whatever reason. I might use
```ts
// personal
website_name@domain.com -> personal.email@gmail.com

// work
website_name@work.domain.com -> work.email@work.com
```

It would be especially helpful to do this using subdomains because if I want to change the designation email for those emails craeted on the subdomain it would just be one click.

In other words, I'm asking that my personal domain be supported by the subdomain feature

**Comment by nguyenkims** [comment]
@alita-moore in order for a domain to support subdomain feature in SimpleLogin, your DNS registrar must support wildcard which isn't the case for most DNS registrars. As a workaround, you can add `work.domain.com` as a custom domain on SimpleLogin. The setup would be a bit faster as you don't have to verify ownership (as you already own `domain.com`).

**Reply by LenKiMo** [reply]
If my DNS registrar support wildcard DNS record, what should I do to enable creating and subdomain for my domain automatically?

**Reply by alita-moore** [reply]
I have confirmed that this works. Which I only need one or two so it's a perfectly fine work around. Would be nice to have the option in case my registrar does support wildcard, though.
